server/auth/read_only_capabilities_switcher.test.ts

```diff
@@ -0,0 +1,133 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Please change this to the following

```
Copyright OpenSearch Contributors
```

Reference this PR: https://github.com/opensearch-project/security-dashboards-plugin/pull/847